docs/components/modeler/desktop-modeler/element-templates/defining-templates.md

```diff
@@ -296,8 +297,22 @@ Configures an [output mapping](../../../../concepts/variables/#output-mappings).
 
 Configures a [task header](../../../bpmn/service-tasks/#task-headers).
 
+#### `zeebe:taskDefinition`
+
+| **Binding `type`**          | `zeebe:taskDefinition`                                                            |
+| --------------------------- | --------------------------------------------------------------------------------- |
+| **Valid property `type`'s** | `String`<br /> `Text`<br />`Hidden`<br />`Dropdown`                               |
+| **Binding parameters**      | `property`: The name of the task definition property. Can be `type` or `retries`. |
+| **Mapping result**          | `<zeebe:taskDefinition [property]="[userInput]" />`                               |
+
+Configures the [task](../../../bpmn/service-tasks/#task-definition) for a service or user task.
+
 #### `zeebe:taskDefinition:type`
 
+:::warning
+`zeebe:taskDefinition:type` is a deprecated binding. Instead, use `zeebe:taskDefinition` with `property=type`.
```

Comment:
Is this a breaking change and does this need to be noted anywhere else?

Reply:
It's not a breaking change yet. The binding is still supported but we recommend using `zeebe:taskDefinition` instead, as we may remove `zeebe:taskDefinition:type` some day in the future. It will show up as deprecated when one writes template JSON.